bodhi/server/tasks/__init__.py

```diff
@@ -143,3 +144,23 @@ def tag_update_builds_task(tag: str, builds: typing.List[str]):
     log.info("Received an order to tag builds for an update")
     _do_init()
     main(tag, builds)
+
+
+@app.task(name="work_on_bugs", autoretry_for=(ExternalCallException,),
```

Comment:
I had to add this `ExternalCallException` because I think we don't want to keep retrying the task if the update is not found in the database and a generic BodhiException is raised.